tests/e2e/utils/mock-requests.js

```diff
@@ -867,4 +867,48 @@ export default class MockRequests {
 			methods
 		);
 	}
+
+	/**
+	 * Fulfills a mock GET request for the tour endpoint.
+	 *
+	 * @param {Object} payload - The payload to be returned in the mock response.
+	 * @return {Promise<void>} Resolves when the mock request is fulfilled.
+	 */
+	async fulfillGetTour( payload, status = 200 ) {
+		await this.fulfillRequest(
+			/\/wc\/gla\/tours(\/.*)?\b/,
+			payload,
+			status,
+			[ 'GET' ]
+		);
+	}
+
+	/**
+	 * Mocks an incomplete tour by fulfilling a GET request with the specified tour data.
+	 *
+	 * @param {string} tourId - The unique identifier of the tour to mock.
+	 * @return {Promise<void>} A promise that resolves when the mock request is fulfilled.
+	 */
+	async mockIncompleteTour( tourId ) {
+		await this.fulfillGetTour(
+			{
+				id: tourId,
+				checked: false,
+			},
```

Comment:
This is pretty close. In a real scenario the response will be as follows:
```suggestion
			{
				"message": "Tour not found"
			},
```